### PR TITLE
Merge settings changes with master

### PR DESCRIPTION
**Pull request by Julesssss**
I wanted to update my local project with the most recent version, work on that, then commit. Somehow I merged the 'RSSDownloadDevelopment' branch with the master. Not sure how to fix this without just merging them again.

Also, for some reason I can only assign one person now :/

### CODE REVIEW

**Comment by amitav13** [comment]
You did it the other way around than you think: you pulled master into your your branch, which is fine. You need to pull master into your branch before you make a pull request though. Even if you're the only one pushing changes, it's a good habit to get into. No harm done in pulling from master.

File: app/src/main/java/infrmr/newsapp/github/com/ifrmr/SettingsActivity.java [hunk 1]
```diff
@@ -1,48 +1,17 @@
...
+import android.support.v7.app.ActionBarActivity;
 
-/**
- * Created by user on 22-Apr-15.
- */
 
-
-
-public class SettingsActivity extends PreferenceActivity implements SharedPreferences.OnSharedPreferenceChangeListener {
+public class SettingsActivity extends ActionBarActivity {
```
Comment (amitav13) [inline app/src/main/java/infrmr/newsapp/github/com/ifrmr/SettingsActivity.java:7]: Settings don't usually have an actionbar do they...?

**Comment by amitav13** [comment]
Nice job on the code overall!

**Comment by Julesssss** [comment]
Not sure what the best practice is, but the settings looked weird as a full screen Activity with no title/ActionBar.

Thanks for the tip, starting to get the hang of group collaboration. The version control integration in Android Studio is fantastic.

**Pull request merged**